design-documents/graph-ql/coverage/catalog/catalog-product-price.graphqls

```diff
@@ -0,0 +1,21 @@
+type ProductPriceRange {
```

Comment:
These schema changes should go in scope of separate story with separate scenarios.

Reply:
this is not a change, this schema already in the mainline

Reply:
Please do not include parts of the schema that already exist in scope of this proposal, it is confusing and will cause problems in the future because of duplication. Just reference any existing types from the new schema.

Reply:
Why is a minimum price required but no maximum price is required? I don't see why either should be required.

Reply:
this not a part of a change that we are doing right now just a fixation mainline schema that we use.